stdlib/public/core/ReflectionMirror.swift

```diff
@@ -204,13 +204,17 @@ public struct _EachFieldOptions: OptionSet {
   ///
   /// If this is not set, the top-level type is required to be a struct or
   /// tuple.
-  public static var classType = _EachFieldOptions(rawValue: 1 << 0)
+  public static var classType: _EachFieldOptions {
```

Comment:
Alternatively, you could make this a `let` and make `_EachFieldOptions` conform to `Sendable`.